src/main/kotlin/net/irisshaders/lilybot/extensions/events/LogUploading.kt

```diff
@@ -167,23 +176,24 @@ class LogUploading : Extension() {
 		}
 	}
 
-	@OptIn(InternalAPI::class)
-	private suspend fun postToHasteBin(text: String): String {
-		val client = HttpClient()
-
-		var response = client.post("https://www.toptal.com/developers/hastebin/documents") {
-			body = text
-		}.content.toByteArray().decodeToString()
-
-		if (response.contains("\"key\"")) {
-			response = "https://www.toptal.com/developers/hastebin/" + response.substring(
-				response.indexOf(":") + 2,
-				response.length - 2
-			)
-		}
+	@kotlinx.serialization.Serializable
+	data class logClass(val success: Boolean, val id: String? = null, val error: String? = null)
+	//setting these to null is necessary in case a value is missing, which would cause an error.
```

Comment:
This is very nitpicky but could you please add a space after the slashes in this comment and the one on line 192